src/services/data.services.ts

```diff
@@ -62,13 +66,25 @@ export default class DataService {
       throw new NotFoundError('run', runId);
     }
 
-    return await prisma.data.create({
+    const createdData = await prisma.data.create({
       data: {
         dataType: { connect: { name: dataType.name } },
         time: unixTime,
         run: { connect: { id: run.id } },
-        value: serverData.value as number
+        value: {
+          createMany: {
+            data: serverData.value.map((value) => ({
+              value: value.value as number,
+              unit: value.unit
```

Comment:
This shouldnt habe a unit right?

Reply:
im pretty sure severdata.ts also has value and units. doesnt seem to break when I remove it though